src/routes/guides/+page.svelte

```diff
@@ -0,0 +1,203 @@
+<script lang="ts">
+  import SectionTitle from "$lib/components/SectionTitle.svelte"
+  import QRCodeIcon from "$lib/images/card-qr-code-icon.svg?dataurl"
+  import DataSchemasIcon from "$lib/images/card-schemas-icon.svg?dataurl"
+  import ArrowIcon from "$lib/images/link-arrow-icon.svg"
+  import Grid from "$lib/components/Grid.svelte"
+  import Breadcrumbs from "$lib/components/Breadcrumbs.svelte"
+  import Title from "$lib/components/Title.svelte"
+  import TableOfContents from "$lib/components/TableOfContents.svelte"
+  import MailIcon from "$lib/images/mail-icon.svg"
+  import SlackIcon from "$lib/images/slack-icon.svg"
+  import Button from "$lib/components/Button.svelte"
+  import { GUIDES } from "./urls"
+
+  const SLACK_URL = "https://slack.ioxio.com"
+  const EMAIL_URL = "mailto:support@ioxio.com"
+
+  const cards = Object.values(GUIDES)
+
+  const secondaryCards = [
+    {
+      title: "Defining Data Products",
+      href: "https://www.youtube.com/watch?v=yPzN04ICsbw",
+      alt: "QR code",
```

Comment:
I was a bit surprised to see the alt be `QR code` for a YouTube video. My first thought was that it's a copy&paste error. Then I looked into it and realized it's an alt for the image, not for the video. Not sure if we should consider renaming it to `imageAlt`, `cardImageAlt` or something such to make it more clear.

I also had a look at the images used on the cards (most of the cards are defined in `src/routes/guides/urls.ts`) and I don't really understand what the pattern is for which icon is used. I don't think the ones using the QR code version have anything particular in common. This is for reference what it looks like:
![image](https://github.com/user-attachments/assets/4e1f9e26-f5ee-4981-91b0-bf5aad29ac46)
Since none of these have anything to do with QR codes, I think I'd suggest to use the other icon for all of them.

Reply:
I thinks as long as it guides - we use guides icon

Reply:
but not for video guides